### PR TITLE
gabedit 2.5.1

### PR DESCRIPTION
**Pull request by dvdsnchzIT**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by dvdsnchzIT** [comment]
Hello,
I failed the url, it's https://downloads.sourceforge.net/project/gabedit/gabedit/Gabedit251/GabeditSrc251.tar.gz, how I update without --force ?
Thanks,

File: Formula/gabedit.rb [hunk 1]
```diff
@@ -1,13 +1,12 @@
 class Gabedit < Formula
   desc "GUI to computational chemistry packages like Gamess-US, Gaussian, etc."
   homepage "https://gabedit.sourceforge.io/"
-  url "https://downloads.sourceforge.net/project/gabedit/gabedit/Gabedit250/GabeditSrc250.tar.gz"
-  version "2.5.0"
-  sha256 "45cdde213a09294bbf2df5f324ea11fc4c4045b3f9d58e4d67979e6f071c7689"
-  revision 2
+  url "https://downloads.sourceforge.net/projects/gabedit/files/gabedit/Gabedit251/GabeditSrc251.tar.gz"
```
Comment (nandahkrishna) [inline Formula/gabedit.rb:4]:
```suggestion
  url "https://downloads.sourceforge.net/project/gabedit/gabedit/Gabedit251/GabeditSrc251.tar.gz"
```

File: Formula/gabedit.rb [hunk 2]
```diff
@@ -1,13 +1,12 @@
...
+  version "2.5.1"
+  sha256 "efcb00151af383f662d535a7a36a2b0ed2f14c420861a28807feaa9e938bff9e"
 
   # Consider switching back to checking SourceForge releases once we can alter
-  # the matched version from `250` to `2.5.0`.
+  # the matched version from `250` to `2.5.1`.
```
Comment (nandahkrishna) [inline Formula/gabedit.rb:9]:
```suggestion
  # the matched version from `250` to `2.5.0`.
```

**Comment by dvdsnchzIT** [comment]
gcc -fno-common  tmp/*.o -o gabedit   `pkg-config gtk+-2.0 --libs`   `pkg-config gtkglext-1.0 --libs` -L/usr/lib -lGL -L/usr/lib -lGLU   -lpthread   -g -lm
  Undefined symbols for architecture arm64:
    "_pango_ft2_get_context", referenced from:
        _glInitFontsUsing in UtilsGL.o
    "_pango_ft2_render_layout", referenced from:
        _gl_pango_ft2_render_layout in UtilsGL.o
       (maybe you meant: _gl_pango_ft2_render_layout)
  ld: symbol(s) not found for architecture arm64
  clang: error: linker command failed with exit code 1 (use -v to see invocation)
  make: *** [gabedit] Error 1

**Pull request closed without merging**